api/census.go

```diff
@@ -0,0 +1,23 @@
+package api
+
+import (
+	"encoding/json"
+	"github.com/ONSdigital/dp-dataset-api/models"
+	"net/http"
+)
+
+// this is the handler for the "/census" endpoint
+func (api *DatasetAPI) getCensus(writer http.ResponseWriter, request *http.Request) {
```

Comment:
naming the ResponseWriter `w` and the Request `req` is much more inline with common conventions

Reply:
ok

Reply:
fixed